### PR TITLE
Use new gridlines from https://github.com/hypar-io/Elements/pull/705

### PR DESCRIPTION
**Pull request by serenayl**
This draws GridLine using the new `GridLine` class, rather than just lines and circles.

---

Building Blocks contains many functions that are used in Hypar examples and in the Hypar tour. Please ensure that that following are true before granting this PR.

- [x] The function has been deployed to dev.
- [ ] The function has been tested against the tour. If the function is not used in the tour, mark this as not applicable (NA).
  - This is still applicable, but we know Structures is pending these changes and so this will not work.

---
This change is [Reviewable](https://reviewable.io/reviews/hypar-io/buildingblocks/69)

### CODE REVIEW

File: Grids/Grid/src/Grid.cs [hunk 1]
```diff
@@ -318,17 +318,19 @@ private static string GetName(string namingPattern, int idx)
...
-
             var lineDir = (line.End - line.Start).Unitized();
             var circleCenter = line.Start - (lineDir * (CircleRadius + lineHeadExtension));
 
-            model.AddElement(new Elements.GridLine(new Polyline(new List<Vector3>() { line.Start, line.End }), null, null, null, false, Guid.NewGuid(), name));
-            model.AddElement(new ModelCurve(new Line(line.Start - (lineDir * lineHeadExtension) + elevation, line.End + elevation), material, name: name));
-            model.AddElement(new ModelCurve(new Circle(circleCenter + elevation, CircleRadius), material));
-            texts.Add((circleCenter + elevation, Vector3.ZAxis, lineDir, name, Colors.Darkgray));
-            return new GridLine(line, name);
+            var gridline = new GridLine();
```
Comment (ikeough) [inline Grids/Grid/src/Grid.cs:324]: Yay!

File: Grids/Grid/src/UtilityClasses.cs [hunk 2]
```diff
@@ -14,15 +14,15 @@ public GridGuide(Vector3 point, string name = null)
         }
     }
 
-    public class GridLine
-    {
-        public Line Line;
-        public string Name;
+    // public class GridLine
```
Comment (ikeough) [inline Grids/Grid/src/UtilityClasses.cs:17]: is this ever going to be used? If not, let's remove it.

File: Grids/Grid/server/Program.cs [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+namespace Hypar.Server
+{
+    public class Program
+    {
+        public static async Task Main(string[] args)
+        {
+            await HyparServer.StartAsync(
+                args,
+                Path.GetFullPath(Path.Combine(@"/Users/serenali/Github/hypar/BuildingBlocks/Grids/Grid/server", "..")),
```
Comment (ikeough) [inline Grids/Grid/server/Program.cs:14]: This is why I've been git ignoring the server project.

**Review by ikeough — CHANGES_REQUESTED** [review]

File: Grids/Grid/server/Program.cs (hunk 3 above)
Comment (andrewheumann) [reply Grids/Grid/server/Program.cs:14]: we moved back to using relative paths for the server project so it wouldn't have any user-specific paths. but I think we should .gitignore the server project by default.

**Review by serenayl — COMMENTED** [review]
> *[Reviewable](https://reviewable.io/reviews/hypar-io/buildingblocks/69)* status: 0 of 14 files reviewed, 3 unresolved discussions (waiting on @andrewheumann and @ikeough)

___
*[Grids/Grid/src/Grid.cs, line 324 at r1](https://reviewable.io/reviews/hypar-io/buildingblocks/69#-Moo3vGT3tgN6uE7CNd5-r1-324:-MqWLIMo9FMOpll1S6bS:b-896fix) ([raw file](https://github.com/hypar-io/buildingblocks/blob/1156531648b6037098706f3f7946bbf867353b46/Grids/Grid/src/Grid.cs#L324)):*
<details><summary><i>Previously, ikeough (Ian Keough) wrote…</i></summary><blockquote>

Yay!
</blockquote></details>

Done.
___
*[Grids/Grid/src/UtilityClasses.cs, line 17 at r1](https://reviewable.io/reviews/hypar-io/buildingblocks/69#-Moo3vGT3tgN6uE7CNd7-r1-17:-MqWLFje8p3oyBPkIiOY:b-896fix) ([raw file](https://github.com/hypar-io/buildingblocks/blob/1156531648b6037098706f3f7946bbf867353b46/Grids/Grid/src/UtilityClasses.cs#L17)):*
<details><summary><i>Previously, ikeough (Ian Keough) wrote…</i></summary><blockquote>

is this ever going to be used? If not, let's remove it.
</blockquote></details>

Done.
___
*[Grids/Grid/server/Program.cs, line 14 at r1](https://reviewable.io/reviews/hypar-io/buildingblocks/69#-Moo3vGT3tgN6uE7CNdC-r1-14:-MqWLGrQDeECpZlLyGc-:b-896fix) ([raw file](https://github.com/hypar-io/buildingblocks/blob/1156531648b6037098706f3f7946bbf867353b46/Grids/Grid/server/Program.cs#L14)):*
<details><summary><i>Previously, andrewheumann wrote…</i></summary><blockquote>

we moved back to using relative paths for the server project so it wouldn't have any user-specific paths. but I think we should .gitignore the server project by default.
</blockquote></details>

Done.

**Review by ikeough — APPROVED** [review]

**Pull request merged**